frontend/main-app/src/components/DeleteForm/DeleteForm.tsx

```diff
@@ -0,0 +1,88 @@
+import classes from "./DeleteForm.module.css";
+import { Button, Modal, Paragraph, Textfield } from "@digdir/design-system-react";
+import React, { FormEvent, ReactNode, useContext, useState } from "react";
+import { TrashFillIcon } from "@navikt/aksel-icons";
+import { FormContext } from "../Dashboard";
+import { getApiUrl } from "../Login/LoginUtils";
+import axios from "axios";
+import { Trans } from "react-i18next";
+import { t } from "i18next";
+
+interface ModalContentProps {
+  className: string;
+}
+
+const ModalContent: React.FC<ModalContentProps> = ({ className }) => {
+  const form = useContext(FormContext);
+  const [formError, setFormError] = useState("");
+
+  const deleteForm = async (formId: string) => {
+    const apiUrl = getApiUrl();
+    const deleteFormEndpoint = `${apiUrl}/api/forms/${formId}`;
+    const deleteSubmissionsEndpoint = `${apiUrl}/api/submissions/form/${formId}`;
+
+    try {
+      await axios.delete(deleteFormEndpoint);
+      await axios.delete(deleteSubmissionsEndpoint);
+      location.reload();
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  const handleDelete = (event: FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    const formData = new FormData(event.currentTarget as HTMLFormElement);
+    const cleanFormData = Object.fromEntries(formData);
+
+    if (cleanFormData.confirmDelete.toString().toLowerCase() === "delete") {
+      deleteForm(form.id.toString());
+    } else {
+      setFormError(t("dashboard.delete.error"));
+    }
+  };
+
+  return (
+    <form className={className} onSubmit={handleDelete}>
+      <Paragraph>{t("dashboard.delete.areYouSure")}</Paragraph>
```

Comment:
What if we update this to "`dashboard.delete.form.confirmation`"🙂

Reply:
Here we have a text divided into two paragraphs.
What about this?

```suggestion
      <Paragraph>{t("dashboard.delete.form.confirm1")}</Paragraph>
      <Paragraph>
        <Trans i18nKey="dashboard.delete.form.confirm2" />
      </Paragraph>
```

```
"dashboard.delete.form.confirm1": "Are you sure you want to delete this form? Both the form and its submissions will be deleted permanently.",
"dashboard.delete.form.confirm2": "To confirm, type <strong>delete</strong> in the box below.",
  
```

Reply:
Maybe the numbers  will not be descriptive enough. 😊, what about ;
```
dashboard.delete.form.confirm
dashboard.delete.form.instructions..confirm
```

Reply:
Good idea!
I landed on this:
```
dashboard.delete.form.confirm.question
dashboard.delete.form.confirm.instruction
```